cypress/integration/milmove/ppms/estimatedIncentive.js

```diff
@@ -44,5 +44,5 @@ function navigateToEstimatedIncentivePage() {
 }
 
 function verifyShipmentSpecificInfo() {
-  cy.get('.container h2').contains('$100');
+  cy.get('.container h2').contains('$10000');
```

Comment:
Looks like it might need a comma

```suggestion
  cy.get('.container h2').contains('$10,000');
```

![image](https://user-images.githubusercontent.com/35938642/160707095-952651f4-1013-466d-a264-aff14a7b1f29.png)

![image](https://user-images.githubusercontent.com/35938642/160707120-4ddc31d9-a539-4f6d-9b19-06844222706f.png)